casper/src/test/scala/coop/rchain/casper/genesis/contracts/RevIssuanceTest.scala

```diff
@@ -45,91 +48,102 @@ class RevIssuanceTest extends FlatSpec with Matchers {
       val runtimeManager = RuntimeManager.fromRuntime(activeRuntime).unsafeRunSync
       val emptyHash      = runtimeManager.emptyStateHash
 
-      val ethAddress      = "0x041e1eec23d118f0c4ffc814d4f415ac3ef3dcff"
-      val initBalance     = 37
-      val wallet          = PreWallet(ethAddress, initBalance)
+      val secKey =
+        Base16.unsafeDecode("a68a6e6cca30f81bd24a719f3145d20e8424bd7b396309b0708a16c7d8000b76")
+      val pubKey = Secp256k1.toPublic(secKey)
+      val prefix = "0x"
+      val ethAddress = prefix + Base16
+        .encode(Keccak256.hash(pubKey.drop(1)))
+        .takeRight(40)
+      val revAddress  = RevAddress.fromEthAddress(ethAddress.drop(2))
+      val initBalance = 100000L
+      val vaults      = List(Vault(revAddress, initBalance))
+
+      // These are unused but are inserted due to require(validators.nonEmpty)
       val (_, validators) = (1 to 4).map(_ => Ed25519.newKeyPair).unzip
       val bonds           = createBonds(validators)
       val posValidators   = bonds.map(Validator.tupled).toSeq
-      val (_, genesisPk)  = Ed25519.newKeyPair
-      val vaults = Traverse[List]
-        .traverse(posValidators.map(_.pk).toList)(RevAddress.fromPublicKey)
-        .get
-        .map(Vault(_, 1000L))
+
+      val (_, genesisPk) = Ed25519.newKeyPair
+
       val genesisDeploys =
         Genesis.defaultBlessedTerms(
           0L,
           ProofOfStake(1L, Long.MaxValue, posValidators),
-          wallet :: Nil,
+          Nil,
           Faucet.noopFaucet,
           genesisPk,
           vaults,
```

Comment:
The custom `vaults` in here mean that this test is testing its own setup rather than production

We should reimplement this test using RhoSpec, along the lines of:

```scala
class RevIssueanceSpec extends RhoSpec(
      CompiledRholangSource("RevIssueanceTest.rho"),
      TestUtil.defaultGenesis.copy(
        wallets = Seq(("0x041e1eec23d118f0c4ffc814d4f415ac3ef3dcff", 42))
      )
      Seq.empty,
      30.seconds
)

//inside RevIssueanceTest.rho:
 - findOrCreate the vault for eth address above
 - get its balance
 - check the balance is as expected
```

Notice the `Genesis` object passed above does not allow for customisation of adding / replacing the genesis deploys - it only allows for parameterizing them, just like the config files do in production setting.

To be clear:
- RhoSpec constructor would start taking a Genesis parameter
- TestUtil would have the Genesis object that's created in `TestUtil.defaultGenesisSetup` exposed as a method/constant